charts/ingress-nginx/values.yaml

```diff
@@ -361,6 +361,8 @@ controller:
       enabled: false
       annotations: {}
 
+      loadBalancerSourceRanges: []
```

Comment:
Please add some comment about the setting

Reply:
added